poetry/utils/exporter.py

```diff
@@ -154,6 +154,8 @@ def _export_requirements_txt(
                 url = (
                     repository.authenticated_url if with_credentials else repository.url
                 )
+                if repository.url.startswith("http://"):
+                    indexes_header += "--trusted-host {}\n".format(url)
```

Comment:
Shouldn't this be hostname and not the url?

Reply:
yup!